deepseg.cc

```diff
@@ -403,21 +450,27 @@ int main(int argc, char* argv[]) {
 
 		if (!debug) { printf("."); fflush(stdout); continue; }
 
-		int e2 = cv::getTickCount();
-		float t = (e2-e1)/cv::getTickFrequency();
-		printf("FPS: %5.2f\r",1.0/t);
-		fflush(stdout);
-		if (debug < 2) continue;
-
 		cv::Mat test;
 		cv::cvtColor(raw,test,CV_YUV2BGR_YUYV);
 		cv::imshow("output.png",test);
 		if (cv::waitKey(1) == 'q') break;
+
+		// throttle main trhead to save performance for tensorflow threads
+		usleep(100000);
```

Comment:
Maybe use condition variables or some event monitoring library like libev to handle the multiplexing …